application-forum-ui/src/main/resources/ForumCode/StyleSheetExtension.xml

```diff
@@ -332,6 +332,10 @@ XWIKI STYLES
   position: relative;
 }
 
+.topic-author img{
```

Comment:
An empty space should be kept before `{`. See the https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/.